seed/models/models.py

```diff
@@ -33,7 +35,21 @@
     (BUILDINGSYNC_RAW, 'BuildingSync Raw'),
     (GREEN_BUTTON, 'GreenButton'),
     (PORTFOLIO_METER_USAGE, 'PM Meter Usage'),
+    (SENSOR_METADATA, 'SensorMetadata'),
```

Comment:
A migration isn't needed because we don't actually reference `SEED_DATA_SOURCES` in our models, right?

Reply:
@nllong We do, but it's _never_ been used: https://github.com/SEED-platform/seed/blob/bugfix/4076-espm-source-type/seed/models/column_mappings.py#L116

The prod db only has null values in that column.  Instead of the 0196 migration in this PR, I'd prefer to just delete that column.

Reply:
Just update the PR to remove the column from the `columnmapping` table, and added enum enforcement to the `propertystate` table